js/objectsBasics.js

```diff
@@ -0,0 +1,550 @@
+/*
+OBJECTS
+    An object is a collection of properties. A property is an 
+    association between a name (or key) and a value.
+    An object is created using the curly braces {...}.
+    An empty object can be created using the two ways:
+*/
+    // 1.
+    let personalDetails = new Object(); // object constructor syntax
+
+    // 2.
+    let userDetails = {};  // object literal syntax
+
+    console.log( typeof(personalDetails) ) // object
+    console.log( typeof(userDetails) ) // object
+
+/*
+    Literals and properties
+        we can use this format to create properties in an object.
+        object = {
+            key: value
+        }
+        and separate the values with a comma.
+        e.g.
+*/
+        let carFeatures = { // object
```

Comment:
There seems to be unnecessary indentation here, similar to some other places of this code. Please visit https://google.github.io/styleguide/jsguide.html#formatting-block-indentation for some guidelines.

Reply:
The tutorial is very helpful. Thanks.